File: final-project-backend-resumaid/src/main/java/nus/iss/edu/sg/final_project_backend_resumaid/model/Booking.java
package nus.iss.edu.sg.final_project_backend_resumaid.model;

import com.google.api.client.util.DateTime;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Booking {

    private String id;
    private String userid;
    private DateTime starttime;
    private DateTime endtime;
    private String meetinglink;
}

File: final-project-frontend-resumaid/src/app/models/booking.ts
export interface BookingSlice {
    booking: any
}

export interface Booking {
    id: string
    userid: string
    starttime: DateTime
    endtime: DateTime
    meetinglink: string
}

export interface DateTime {
    value: EpochTimeStamp
    dateOnly: boolean
    timeZoneShift: number
}